### DISCUSSION TITLE
Cannot read properties of undefined (reading 'lib')

### DISCUSSION THREAD

**Discussion by ghost** [diegomura/react-pdf#1618] [Q&A]
i just imported updated version of react-pdf  and its showing me this error 
https://codesandbox.io/s/silly-jones-p27br?file=/src/App.js

![image](https://user-images.githubusercontent.com/53328265/143547278-a71989d1-9b3c-4ef7-85d5-d091432b0e3d.png)

Cannot read properties of undefined (reading 'lib')... i didn't made any changes in code its same as in documentation

**Comment by jeetiss** [comment]
codesanbox fails to build `crypto-js/pbkdf2` for some reason. 

https://codesandbox.io/s/romantic-dawn-sci6h?file=/src/App.js

I change the `crypto-js` imports here https://github.com/diegomura/react-pdf/pull/1607 and hope when the pr will be released the problem were gone

**Reply by ghost** [reply]
yes later i found that issues and  fixed it on my  local repo... hope the PR will release soon..  BTW good work ...
